tests/dags/test_job_operator.py

```diff
@@ -15,6 +15,7 @@
 
 envs = {
     "PASS_ARG": "a test",
+    "JINJA_ENV": "{{ ds }}",
```

Comment:
Why would you need templated environment variables?

Reply:
Default in airflow. In general also, you might need the current task execution date, other task info params, or params sent to the dag in the json variables. This would be best controlled through the envs. Please remember that you can run the operator without a yaml definition, by sending the image and a command.